src/main/java/org/jenkinsci/plugins/scriptsecurity/scripts/ClasspathEntry.java

```diff
@@ -58,10 +58,18 @@ public ClasspathEntry(@Nonnull String path) throws MalformedURLException {
     }
     
     static URL pathToURL(String path) throws MalformedURLException {
+        if (path.isEmpty()) {
```

Comment:
Well, spaces would imply that the user actually deliberately typed something in the textbox, which would be a different case.